Sources/HKLogger/Common/HKLoggerType.swift

```diff
@@ -27,4 +27,43 @@ public extension HKLoggerType {
         }
     }
     
+    /// Message generation to replace log message
+    var message: String? {
+        switch self {
+        case .networking(let request, let response, let body):
+            return getMessageFor(request: request, response: response, body: body)
+        default:
+            return nil
+        }
+    }
+}
+
+private extension HKLoggerType {
+    func getMessageFor(request: URLRequest, response: HTTPURLResponse, body: Data?) -> String? {
```

Comment:
Maybe we can add a test for this function

Reply:
Test added